packages/tokens/.babelrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "plugins": ["annotate-pure-calls"]
+}
```

Comment:
I assume we need this due to using Jest...

I don't see any tests in this package...are they elsewhere? If there aren't any tests...do we need to include babel, Jest, and other dev deps for this package?